2023-risks/M6: Draft-Insufficient-Input-Output-Validation.md

```diff
@@ -0,0 +1,128 @@
+---
+
+layout: col-sidebar
+title: "M6: Insufficient Input/Output Validation"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+If the application fails to properly validate and sanitize data received from external sources like user inputs or network data. This oversight can lead to various security vulnerabilities, such as SQL injection, Command Injection or cross-site scripting (XSS) attacks. 
+
+These attacks can lead to unauthorized data access, manipulation, and potentially compromise the entire system. 
```

Comment:
Maybe replace the entire system with something mobile-specific. This wording seems like it is talking about backends.

Reply:
Yes, backend system or maybe we can replace it with Database?

Reply:
review pls.